Assignment 2.Rmd

```diff
@@ -81,20 +101,137 @@ plot(g,layout=layout.fruchterman.reingold, vertex.color=VERTEX$gender)
 
 plot(g,layout=layout.fruchterman.reingold, vertex.color=VERTEX$gender, edge.width=EDGE$count)
```

Comment:
Did you delete some data cleaning steps? Seems there is no "VERTEX" or "EDGE" in the code which you use in the plot command.